locales/pl/pl.json

```diff
@@ -35,7 +35,7 @@
     "All rights reserved.": "Wszelkie prawa zastrzeżone",
     "Already registered?": "Zarejestrowany już?",
     "American Samoa": "Samoa Amerykańskie",
-    "An error occured while uploading the file.": "Wystąpił błąd podczas przesyłania pliku.",
+    "An error occurred while uploading the file.": "Wystąpił błąd podczas przesyłania pliku.",
```

Comment:
Don't update tags please

Reply:
I'm only asking: it's correct with one `r`, or this is typo?
Google Translate see it as typo:
![image](https://user-images.githubusercontent.com/42643563/140727346-4e97ba0c-951a-4707-9d54-a7298e620ef5.png)
Only asking ;]

Reply:
It's not correct but the original file has the error. It's just a tag, so it's not important

Reply:
So from where that original file comes from? I'll report that to author.

Reply:
This PR must be sent to Laravel, if they correct it there, we correct us here!